example/config/README.md

```diff
@@ -0,0 +1,249 @@
+# Configuration
```

Comment:
Could you link to this file from the documentation site navigation in `docs/mkdocs.yaml`? The location you put it is useful, I think, because that is probably a commonly viewed directory, but I'm sure people looking through the site docs would also be interested in this.